plugins/source/facebookmarketing/.gitignore

```diff
@@ -0,0 +1,23 @@
+# Binaries for programs and plugins
+*.exe
+*.exe~
+*.dll
+*.so
+*.dylib
+
+# Test binary, built with `go test -c`
+*.test
+
+# Output of the go coverage tool, specifically when used with LiteIDE
+*.out
+
+# Dependency directories (remove the comment below to include it)
+# vendor/
+
+
+.DS_Store
+
+facebookmarketing
```

Comment:
Nice one of ignoring it, but you should also remove the binary as it's committed at the moment